decidim-core/app/helpers/decidim/application_helper.rb

```diff
@@ -2,5 +2,8 @@
 module Decidim
   # Main module to add application-wide helpers.
   module ApplicationHelper
+    def link_to_organization(organization)
+      link_to(organization.name, "http://#{organization.host}")
```

Comment:
I think this can be refactored into:

```ruby
link_to(organization.name, root_url(host: organization.host))
```

This way it keeps the protocol & stuff.